packages/token-lists/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pancakeswap/token-lists",
-  "version": "2.14.4",
+  "version": "2.14.5",
```

Comment:
patch version can be 0 for 2.14.x

Reply:
also its should be updated to 2.**15** probably due to too many changes

Reply:
actually can be 14 for now since top-100 missed the previous bump (separate version handling is coming soon, I promise)